resource-managers/kubernetes/docker/src/main/dockerfiles/spark/decom.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+set -ex
+export LOG=/dev/termination-log
```

Comment:
`/dev/` looks like a very weird place for a log file.

In fact, is this log file useful at all? Won't it go away as soon as the container stops? (I looked at the k8s page around these event handlers but it doesn't seem to explain where the output of these commands end up.)

Reply:
Yeah I agree on /dev being weird. We can 86 this I found it useful during debugging to get the termination information